src/responder/kcm/kcmsrv_ccache_secdb.c

```diff
@@ -873,24 +873,25 @@ static errno_t ccdb_secdb_get_cc_for_uuid(TALLOC_CTX *mem_ctx,
         cli_cred.ucred.gid = pwd->pw_gid;
 
         ret = key_by_uuid(tmp_ctx, secdb->sctx, &cli_cred, uuid, &secdb_key);
-        if (ret == ENOENT) {
-            ret = EOK;
-            goto done;
-        } else if (ret != EOK) {
+        if (ret != EOK) {
```

Comment:
This case is an exception from commit description.
I don't know if special handling of `ENOENT` here makes any sense (actually, this should indicate a serious internal DB corruption and thus probably it is safer to stop here).
But in case case it was wrong to return `EOK` here because output arg `_cc_list` was left uninitialized.